PowerShellSyntax.tmLanguage

```diff
@@ -328,7 +328,7 @@
 		</dict>
 		<dict>
 			<key>match</key>
-			<string>(?&lt;!\w)-(?i:[ic]?(?:eq|ne|[gl][te]|(?:not)?(?:like|match|contains|in)|replace))(?!\p{L})</string>
+			<string>(?&lt;!\w)-(?i:[ic]?(?:eq|ne|[gl][te]|(?:not)?(?:like|match|contains|in)|any|all|replace))(?!\p{L})</string>
```

Comment:
I think you mean this, as this will only apply `-all` and `-any`.
```suggestion
			<string>(?&lt;!\w)-(?i:(?:[ic]?(?:eq|ne|[gl][te]|(?:not)?(?:like|match|contains|in)|replace))|any|all)(?!\p{L})</string>
```